edgelet/iotedge/src/check/mod.rs

```diff
@@ -887,7 +929,50 @@ mod tests {
         }
 
         match WellFormedConnectionString::default().execute(&mut check, &mut runtime) {
-            CheckResult::Warning(err) => assert!(err.to_string().contains("Device not configured with manual provisioning, in this configuration 'iotedge check' is not able to discover the device's backing IoT Hub.")),
+            CheckResult::Ok => {
+                assert_eq!(check.iothub_hostname, Some(hub_name.to_owned()));
+            }
+            check_result => panic!("parsing {} returned {:?}", filename, check_result),
```

Comment:
You may want to change this message to something else, so that it's clear from the panic log whether it was this check or the previous one that failed.